Expensify/App
# Add Logs to ReactNativeChat / Ion

cc @flodnv @AndrewGable 

This PR adds basic logging to our main log server for ReactNativeChat.

Must be tested in connection with [this Web-E PR](https://github.com/Expensify/Web-Expensify/pull/29468) so we can verify the logs are showing up with the correct source

# Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/144084

### Tests
1. Add a `Log.info('test', true);` in `Expensify.js`
1. `tail.sh` + grep for `expensifyCash`
1. Refresh the web app and verify that logs are showing with the correct platform and version
1. Test on various platforms and make sure the platform updates

**Example log line:**
```
2020-10-23T21:03:12.389699+00:00 expensidev php-fpm: xL0IZK /api.php we@dont.know !expensifyCash[web]1.0.1-144! ?api? [info]
```

### Screenshots
❌
